Assets/Scripts/BlockScripts/Letter.cs

```diff
@@ -0,0 +1,19 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class Letter 
+{
+    public string letter = string.Empty;
+    public int engWeight = 0;
```

Comment:
Can probably get away with a single weight and value for a letter. If a user changes languages we can generate new letters with different weights.

Reply:
It'll make adding new languages easier